src/pystage/core/_sensing.py

```diff
@@ -33,7 +33,7 @@ def sensing_mousey(self):
 
     def sensing_loudness(self):
         # See events area, not sure if we support microphone access
-        pass
+        return -1
```

Comment:
Is this change necessary for the "list variables" feature?

Reply:
It's neccessary for showing loudness on screen. Same with Scratch.

Reply:
So, it absolutely does not belong in this PR.